crates/turbopack-css/src/references/import.rs

```diff
@@ -345,11 +345,12 @@ impl ImportAssetReference {
 impl ModuleReference for ImportAssetReference {
     #[turbo_tasks::function]
     fn resolve_reference(&self) -> Vc<ModuleResolveResult> {
-        css_resolve(
+        url_resolve(
             self.origin,
             self.request,
-            Value::new(CssReferenceSubType::AtImport),
+            Value::new(UrlReferenceSubType::CssAtImport),
             Some(self.issue_source),
+            IssueSeverity::Error.cell(),
         )
```

Comment:
I wonder that this works... Doesn't this reference the imported CSS without processing it as CSS?